### PR TITLE
Tinc-devel updated to current

### PR DESCRIPTION
**Pull request by joshin42**
( This is my first push request. Let me know if I'm doing it wrong. )

Updated Tinc-devel to 1.1.pre16

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.x
Xcode 8.x

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: net/tinc/Portfile [hunk 1]
```diff
@@ -43,8 +43,8 @@ post-destroot {
...
     conflicts       tinc
 
-    version         1.1pre14
+    version         1.1pre16
     revision        0
-    checksums       rmd160  25f1edbf475a33e82527d62e40ba73b607aecd60 \
-                    sha256  e349e78f0e0d10899b8ab51c285bdb96c5ee322e847dfcf6ac9e21036286221f
+    checksums       rmd160  29a7953700515cc553c12662d0e93dd87c0a46c0 \
+                    sha256  9934c53f8b22bbcbfa0faae0cb7ea13875fe1990cce75af728a7f4ced2c0230b
 }
```
Comment (mf2k) [inline net/tinc/Portfile]: The `size` parameter needs to be added here.

**Comment by mf2k** [comment]
Update your commit message to be more specific. Like this:

`tinc-devel: Update to version 1.1.pre16`

**Comment by mf2k** [comment]
Also, remove the .DS_Store files from the commit.

**Comment by pmetzger** [comment]
Oh, and a reminder: when you update those things, squash the commits and force push. As you (unfortunately) used your master branch for this, it is _very_ important that you not "git pull" until you've finished getting this merged.

**Comment by joshin42** [comment]
Thanks for the feedback.

Given that I'm new to Git as a committer, and OS X /  MacPorts as a user, I'm not sure how do most of what is being requested, and might have bitten off more than I'm willing to chew in submitting the pull request..  

There's no size parameter in the original file. How would I know to add that? The more difficult you make it for someone to come up the learning curve, the less likely someone is going to bother.

May I suggest that you tag all files missing parameters with comments stating any missing parameters and giving an example of what they should look like. 

As to figuring out how to squash commits, removing the .DS_Store file, and not using the master branch, sorry, but I'm not going to spend 50x the time figuring out how to do that than it took to do the commit. 

Again, I appreciate the feedback, and if I choose to try this again, I'll spend that time to come up the learning curve. But as of right now, I'm just dipping a toe in the OS X world to see if it's worth the effort.

**Comment by pmetzger** [comment]
> Given that I'm new to Git as a committer, and OS X / MacPorts as a user, I'm not sure how do most of what is being requested, and might have bitten off more than I'm willing to chew in submitting the pull request..

Understanding how to use git and github is a general skill and has very little to do with MacPorts itself as such. I suggest this book: https://git-scm.com/book/en/v2 as a start.

For a lot of open source projects, git tends to be the VCS in use, and people expect that when pull requests come in, there will be comments and requests for changes. Understanding how to make changes in your fork of the repo and how to push those updated patches to the maintainers is a bit of a basic skill.

> There's no size parameter in the original file. How would I know to add that?

We told you, that's part of what review of a pull request is for. The documentation explains that this is part of what we expect.`port lint` will also tell you these days, and the manual tells you.

> May I suggest that you tag all files missing parameters with comments stating any missing parameters and giving an example of what they should look like.

I'm not sure what this means.

**Pull request closed without merging**